pkg/plugins/golang/v4/scaffolds/internal/templates/webhooks/webhook.go

```diff
@@ -81,7 +81,7 @@ func (f *Webhook) SetTemplateDefaults() error {
 	if f.Force {
 		f.IfExistsAction = machinery.OverwriteFile
 	} else {
-		f.IfExistsAction = machinery.Error
+		f.IfExistsAction = machinery.SkipFile
```

Comment:
Is this to enable executing conversion command multiple times? Since this template covers both validation and defaulting - there could be issues with just skipping file generation. Do we want to add print statements?

Reply:
You are right here.
We have a terrible UX now to work with webhooks since is not possible create them without force as a follow up.
we need to fix that as well:  https://github.com/kubernetes-sigs/kubebuilder/issues/4146

I will think a solution here for we not be blocked by: https://github.com/kubernetes-sigs/kubebuilder/issues/4146
until we have this one.